packages/components/organisms/f-checkout/src/components/Guest.vue

```diff
@@ -8,7 +8,7 @@
             :aria-invalid="isFirstNameEmpty"
             name="guest-first-name"
             maxlength="100"
-            :label-text="$t('guest.firstName')"
+            :label-text="$t('formFields.customer.firstName.label')"
```

Comment:
Could we use some variables so that we don't need to type out the full qualifications each time?

Reply:
This will be done in the next pr https://github.com/justeat/fozzie-components/compare/f-checkout-translations...f-checkout-guest-form-field?expand=1